lib/relation.gi

```diff
@@ -696,110 +696,7 @@ InstallMethod(StronglyConnectedComponents, "for general binary relations",
         local r,        # representation of rel as a binary relation on points
               e,        # Equivalence relation representation of the
                         # predecessor subgraph
-              s,        # Sorted list of the source of rel
-              DFS,      # Depth first search functions
-              DFSVisit,    ## Recursive
-              DFSVisitNR,  ## Non Recursive
-                        # Global variables for DFS
-              time,     # time
-              color,    # 0=white, 1=gray, 2=black
-              adj,      # adjacency list i.e. successors of rel
-              pi,       # predecessor subgraph
-              dtime,    # discover time
-              ftime;    # finish time
-
-        ## Recursive version of a depth first visit
-        ##
-        DFSVisit := function(u)
```

Comment:
So there is a bunch of dead code being removed here, and I am a bit torn about that: on the one hand, it is dead, and it's good we recognized that. And it'll likely always stay dead and so we can remove it... On the other hand, I really wonder what the story here is. Was this code simply superseded by something better? Or was there a bug and it was disabled? Or what else is going on? <ight try to dig into `git blame` to find out who wrote this...

Reply:
I think it's 

> Was this code simply superseded by something better? 

I'll double check and write some more in a moment.

Reply:
I think it's completely safe to remove these lines of code, the method is to compute the strongly connected components of a binary relation, it looks like the old (removed) code did this using a DFS, and the newer code uses the kernel function `STRONGLY_CONNECTED_COMPONENTS_DIGRAPH`.